comp/remote-config/rcclient/rcclientimpl/rcclient.go

```diff
@@ -161,60 +161,76 @@ func (rc rcClient) start() {
 }
 
 func (rc rcClient) mrfUpdateCallback(updates map[string]state.RawConfig, applyStateCallback func(string, state.ApplyStatus)) {
-	var failover *bool
+	// If the updates map is empty, we should unset the failover settings if they were set via RC previously
+	if len(updates) == 0 {
+		// Determine the current source of the multi_region_failover.failover_metrics and multi_region_failover.failover_logs cfg value
```

Comment:
Nit: The code describes this sufficiently.

Reply:
deleted in [fbffb0a](https://github.com/DataDog/datadog-agent/pull/24754/commits/fbffb0a537975d05cdb1303171399747a2fab506) 👍